ref: Add `plots.auto_open` in `dvc.config` (#3101)
iterative/dvc.org: content/docs/command-reference/config.md

  [custom HTML template](/doc/command-reference/plots#html-templates) for
  `dvc plots`. Accepts a path relative to the `.dvc/` folder.

> - `plots.auto_open` - if enabled, DVC will automatically open a browser with the
>   `html` generated by `dvc plots` commands. Accepts values `true` and `false`
>   (default).

Reviewer: ```suggestion
- `plots.auto_open` - if `true`, DVC will automatically open the HTML
  generated by `dvc plots` commands in a browser. `false` by default
```

  [custom HTML template](/doc/command-reference/plots#html-templates) for
  `dvc plots`. Accepts a path relative to the `.dvc/` folder.

- `plots.auto_open` - if `true`, DVC will automatically open the HTML

>   generated by `dvc plots` commands in a browser. `false` by default

Reviewer: Q: Will this apply to `dvc exp show --html` (or whatever the final command for PCPs is) ? If so maybe add a check box to update this in https://github.com/iterative/dvc.org/pull/3105 at some point).

Reviewer: Also this implies a small update in https://github.com/iterative/dvc.org/pull/3098/files#diff-777292f659a766aaad0a04f44b3ed6eb55770470d3fc8dd3490be2b994e98d68R110 (that's on me)...

Reviewer: So now that https://github.com/iterative/dvc.org/pull/3098 is merged maybe we should address this here?

> - `plots.auto_open` - if `true`, DVC will automatically open the HTML generated
>   by `dvc plots` commands in a browser. `false` by default

Reviewer: Oops another CE
```suggestion
- `plots.auto_open` - if `true`, DVC will automatically open the HTML file
  generated by `dvc plots` commands in a browser. `false` by default
```
